ibis/__init__.py

```
@@ -37,7 +37,7 @@
 
 with suppress(ImportError):
```

Can you try to temporary remove this with? It may be hiding the actual error.

---

Done. New commit pushed to the PR.